src/main/java/org/folio/rest/impl/DelayedTasksAPI.java

```diff
@@ -31,4 +36,53 @@ public void deleteDelayedTaskExpiredMessages(String expirationDate, String statu
       .otherwise(this::mapExceptionToResponse)
       .onComplete(resultHandler);
   }
+
+  @Override
+  public void getDelayedTaskRetry(Map<String, String> okapiHeaders,
+    Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
+
+    RetryEmailsContext retryEmailsContext = new RetryEmailsContext();
+
+    Future.succeededFuture()
+      .compose(v -> lookupConfig(okapiHeaders))
+      .compose(this::checkConfiguration)
+      .map(retryEmailsContext::setConfiguration)
+      .compose(v -> findEmailsForRetry())
+      .compose(this::mapJsonObjectToEmailEntries)
+      .map(retryEmailsContext::setEmailEntries)
+      .compose(context -> CompositeFuture.all(context
+        .getEmailEntries()
+        .getEmailEntity()
+        .stream()
+        .map(email -> sendEmail(context.getConfiguration(), email)
+          .compose(this::updateEmail))
+        .collect(Collectors.toList())))
+      .map(Email.PostEmailResponse::respond200WithTextPlain)
```

Comment:
```suggestion
      .map(Email.PostEmailResponse::respond204WithTextPlain)
```

Reply:
Currently we have such implementation 
/email:
  type:
    collection-item-get:
      schema: emailEntries
      exampleItem: !include examples/email.sample
  post:
    description: Send email notifications
    body:
      application/json:
        schema: emailEntity
    responses:
      200:
        description: "success"
        body:
          text/plain:
            example: "200 Ok"
      400:
        description: "Bad request"
        body:
          text/plain:
          
I can add 204 in email.raml file either change 200 into 204. What would better?